ipatests/test_integration/test_otp.py

```diff
@@ -398,6 +460,106 @@ def test_2fa_disable_single_prompt(self):
             self.master.run_command(['semanage', 'login', '-D'])
             sssd_conf_backup.restore()
 
+    def test_2fa_only_with_password(self):
+        """Test ssh with 2FA only with the password(first factor) when
+        user-auth-type is opt and password.
+
+        Test for :  https://github.com/SSSD/sssd/pull/7500
+
+        Add the IPA user and user-auth-type set to opt and password.
+        Authenticate the user only with password, just press enter
+        at `Second factor`
+        """
+        master = self.master
+        USER3 = 'sshuser3'
+        sssd_conf_backup = tasks.FileBackup(master, paths.SSSD_CONF)
+        first_prompt = 'Enter first factor:'
+        second_prompt = 'Enter second factor:'
+        add_contents = textwrap.dedent('''
+            [prompting/2fa/sshd]
+            single_prompt = False
+            first_prompt = {0}
+            second_prompt = {1}
+            ''').format(first_prompt, second_prompt)
+        set_sssd_conf(master, add_contents)
+        tasks.create_active_user(master, USER3, PASSWORD)
+        tasks.kinit_admin(master)
+        master.run_command(['ipa', 'user-mod', USER3, '--user-auth-type=otp',
+                            '--user-auth-type=password'])
+        try:
+            otpvalue = "\n"
+            tasks.clear_sssd_cache(self.master)
+            github_ticket = "https://github.com/SSSD/sssd/pull/7500"
+            sssd_version = tasks.get_sssd_version(master)
+            with (xfail_context(sssd_version < tasks.parse_version(
```

Comment:
I would write something like:
```
fedora_fail = osinfo.id == 'fedora' and sssd_version < tasks.parse_version("2.10.0")
rhel_fail = osinfo.id == 'rhel' and sssd_version < tasks.parse_version("2.9.5")
with xfail_context(fedora_fail or rhel_fail, reason=github_ticket):
...
```

The sssd copr repo has been fixed and you should be able to retry the test now.